openvmm/openvmm_entry/src/cli_args.rs

```diff
@@ -591,7 +591,15 @@ pub enum DiskCliKind {
     // file:<path>
     File(PathBuf),
     // blob:<type>:<url>
-    Blob { kind: BlobKind, url: String },
+    Blob {
+        kind: BlobKind,
+        url: String,
+    },
+    // crypt:<key_file>:<kind>
+    Crypt {
+        key_file: PathBuf,
```

Comment:
it's not a super critical CLI, but maybe its still worth requiring specifying the key algo here (even though we only currently support one?)